airbyte-webapp/src/hooks/services/Connection/ConnectionFormService.test.tsx

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Is it just the two lines where we weren't able to get away without the `any`?  Could we just disable eslint for those lines?

Reply:
Those 6 lines yeah. But I feel way less strongly about using `any` in a test file over real code. I could move the ignore down if you'd like!